File: HRInventories/HRInventories/Models/Product.cs
using System;
using System.Collections.Generic;

namespace HRInventories.Models
{
    public partial class Product
    {
        public Product()
        {
            Podetail = new HashSet<Podetail>();
            Requestdetail = new HashSet<Requestdetail>();
        }

        public int Productid { get; set; }
        public long Categoryid { get; set; }
        public string Productname { get; set; }
        public string Productdescription { get; set; }
        public string Userid { get; set; }
        public DateTime Createddate { get; set; }
        public string Isdeleted { get; set; }

        public virtual Catagory Category { get; set; }
        public virtual ICollection<Podetail> Podetail { get; set; }
        public virtual ICollection<Requestdetail> Requestdetail { get; set; }
    }
}

File: hrInventoryUi/src/app/shared/models/product.ts
export interface product {
  Productid:number,
  Productname:string;
  Productdescription:string;
  Categoryid:string;
  Userid:string;
  Createddate:Date;
  Isdeleted:string
      
}

export class showOnGrid{
  productid: number;
  categoryid: number;
  productname: string;
  productdescription: string;
  userid: number;
  createddate: Date;
  isdeleted: string;
  balance: number;
  categoryname: string;
}